webextension/js/background/tabPageCounter.js

```diff
@@ -0,0 +1,64 @@
+const tabPageCounter = {
```

Comment:
Will do later I wasn't sure if we wanted to account for opt in telemetry at any point. Was a little tricky to reengineer this and other telemetry bits (this was pre decision to drop it totally that I did the work) So I would rather leave it in as totally non functioning for now. We could *consider* supporting telemetry for people who want to help us also.